.github/workflows/ci.yml

```
@@ -0,0 +1,74 @@
+      uses: gittools/actions/gitversion/execute@v3.0.3
+      with:
+        useConfigFile: true
+        updateAssemblyInfo: true
+    
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v4
+      with:
+        dotnet-version: '9.0.x'
+
+    - name: Cache NuGet packages
+      uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)